MediaBrowser.Providers/Plugins/MusicBrainz/MusicBrainzAlbumProvider.cs

```diff
@@ -256,6 +256,71 @@ public async Task<MetadataResult<MusicAlbum>> GetMetadata(AlbumInfo info, Cancel
             }
         }
 
+        if (Plugin.Instance != null && Plugin.Instance.Configuration.GetMissingTrackInfo)
+        {
+            if (result.Item.Children.Any(s => s is Audio && s.IndexNumber == null))
+            {
+                var rspobj = _musicBrainzQuery.LookupRelease(Guid.Parse(releaseId ?? string.Empty), Include.Recordings);
```

Comment:
String.empty doesn't make much sense here.

Reply:
`String.Empty` is there because the project's StyleCop settings are causing this:

![image](https://github.com/jellyfin/jellyfin/assets/1653992/e16ff020-88a5-42c4-897c-30532c269e4e)

That's the only reason I've done this, it prevents the build otherwise

Reply:
It doesn't make sense to look up an empty id. Check that it isn't null before you try to parse it